Exiled.API/Features/Roles/SpectatorRole.cs

```diff
@@ -66,6 +66,26 @@ public Player SpectatedPlayer
             }
         }
 
+        /// <summary>
+        /// Gets the Roll Rotation of the <see cref="Player"/>.
+        /// </summary>
+        public float RollRotation => Internal.RollRotation;
+
+        /// <summary>
+        /// Gets the vertical rotation of the <see cref="Player"/>.
+        /// </summary>
+        public float VerticalRotation => Internal.VerticalRotation;
+
+        /// <summary>
+        /// Gets the horizontal rotation of the <see cref="Player"/>.
+        /// </summary>
+        public float HorizontalRotation => Internal.HorizontalRotation;
+
+        /// <summary>
+        /// Gets the camera position of the <see cref="Player"/>.
+        /// </summary>
+        public Vector3 CameraPosition => Internal.CameraPosition;
```

Comment:
WHy are all of these in the SpectatorRole class, the one role where none of these values mean anything?

Reply:
yamato told me to add them

Reply:
I removed them